client/my-sites/plugins/plugin-navbar-header/index.jsx

```diff
@@ -0,0 +1,662 @@
+												<li>
+													<a
+														role="menuitem"
+														className="x-dropdown-link x-link"
+														href="//wordpress.com/create-blog/"
+														title={ translate( 'Create a Blog' ) }
+														tabIndex="-1"
+													>
+														{ translate( 'Create a Blog' ) }
+													</a>
+												</li>
+												<li>
+													<a
+														role="menuitem"
+														className="x-dropdown-link x-link"
+														href="//wordpress.com/professional-email/"
+														title={ translate( 'Professional Email' ) }
+														tabIndex="-1"
+													>
+														{ translate( 'Professional Email' ) }
+													</a>
+												</li>
+											</ul>
+											<div className="x-dropdown-content-separator"></div>
+											<ul>
+												<li>
+													<a
+														role="menuitem"
+														className="x-dropdown-link x-link"
+														href="//wordpress.com/p2/?ref=wpcom-product-menu"
+														title={ translate( ' P2: WordPress for Teams' ) }
+														tabIndex="-1"
+													>
+														{ translate( 'P2: WordPress for Teams' ) }
+													</a>
+												</li>
+												<li>
+													<a
+														role="menuitem"
+														className="x-dropdown-link x-link"
+														href="https://wpvip.com/?utm_source=WordPresscom&utm_medium=automattic_referral&utm_campaign=top_nav"
+														title={ translate( ' Enterprise' ) }
+														tabIndex="-1"
+													>
+														{ translate( 'Enterprise' ) }
+														<svg
+															width="9"
+															height="9"
+															xmlns="http://www.w3.org/2000/svg"
+															className="x-icon x-icon--external"
+															role="img"
+															aria-hidden="true"
+															focusable="false"
+														>
+															<path d="M5.5 0v1h1.795L2.38 5.915l.705.705L8 1.705V3.5h1V0H5.5zM8 8H1V1h3V0H1a1 1 0 00-1 1v7a1 1 0 001 1h7a1 1 0 001-1V5H8v3z"></path>
+														</svg>
+													</a>
+												</li>
+											</ul>
+										</div>
+									</li>
+									<li className="x-nav-item x-nav-item--wide">
+										<button
+											role="menuitem"
+											className="x-nav-link x-link"
+											data-dropdown-trigger="features"
+											aria-haspopup="true"
+											aria-expanded="false"
+										>
+											{ translate( 'Features' ) }{ ' ' }
+											<span className="x-nav-link-chevron" aria-hidden="true"></span>
+										</button>
+										<div
+											className="x-dropdown-content"
+											data-dropdown-name="features"
+											role="menu"
+											aria-label={ translate( ' Features' ) }
+											aria-hidden="true"
+										>
+											<ul>
+												<li>
+													<a
+														role="menuitem"
+														className="x-dropdown-link x-link"
+														href="//wordpress.com/features/"
+														title={ translate( ' Features' ) }
```

Comment:
The leading whitespaces in translated strings are quite common in this component, is that intentional?